src/handlers/gestureHandlerCommon.ts

```diff
@@ -180,3 +181,17 @@ export function findNodeHandle(
   if (Platform.OS === 'web') return node;
   return findNodeHandleRN(node);
 }
+
+let scheduledHandlersQueueFlush:
+  | ReturnType<typeof requestAnimationFrame>
```

Comment:
In fact, it's just a `number`, but I like the fact that your approach will work even if they change it to something else.

Reply:
I learned the hard way that `setTimeout` and `setInterval` have different return types on web and in RN. I assumed this would also be the case here. I think it's not much of a problem, it helps with understanding what this variable holds (I think) and will get compiled out anyways.

Reply:
I agree.